app/Commands/StatusCommand.php

```
@@ -0,0 +1,420 @@
+                'enhancement' => count(array_filter($openIssues, fn($i) => in_array('enhancement', $i['labels']))),
+                'bug' => count(array_filter($openIssues, fn($i) => in_array('bug', $i['labels']))),
+                'other' => count(array_filter($openIssues, fn($i) => empty(array_intersect(['epic', 'enhancement', 'bug'], $i['labels'])))),
+            ],
+            'stale_issues' => count(array_filter($openIssues, fn($i) => strtotime($i['updated_at']) < strtotime('-30 days'))),
+            'unassigned_count' => count(array_filter($openIssues, fn($i) => empty($i['assignee']))),
+            'recent_activity' => count(array_filter($openIssues, fn($i) => strtotime($i['updated_at']) > strtotime('-7 days'))),
+            'issues' => $openIssues,
+        ];
+    }
+
+    private function getPullRequestsAnalysis(string $owner, string $repo): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$owner'. (Unused Code Rules)

(UnusedFormalParameter)